careers/careers/utils.py

```diff
@@ -14,10 +14,10 @@ def order_by_function(x):
         order_by = order_by_function
 
     return (
-        sorted(set(
+        sorted(
```

Comment:
Should this change and the one for `test_utils.py` be a part of this PR? Seems unrelated.